cmd/podmanV2/report/templates.go

```diff
@@ -19,13 +20,40 @@ var defaultFuncMap = template.FuncMap{
 	"humanDuration": func(t int64) string {
 		return units.HumanDuration(time.Since(time.Unix(t, 0))) + " ago"
 	},
+	"humanDurationFromTime": func(t time.Time) string {
+		return units.HumanDuration(time.Since(t)) + " ago"
+	},
 	"humanSize": func(sz int64) string {
 		s := units.HumanSizeWithPrecision(float64(sz), 3)
 		i := strings.LastIndexFunc(s, unicode.IsNumber)
 		return s[:i+1] + " " + s[i+1:]
 	},
 	"join":  strings.Join,
 	"lower": strings.ToLower,
+	"numCons": func(cons []*entities.ListPodContainer) int {
```

Comment:
As a TODO I would rather see the entities targeted formatting in a separate map and then merged using AppendFuncMap().  Having Pod formatting available to Images is a code smell.

Reply:
agree! done now